src/pydargs/__init__.py

```diff
@@ -196,6 +229,33 @@ def _add_arguments(parser: ArgumentParser, tp: Type[Dataclass], prefix: str = ""
     return parser
 
 
+def _add_subparsers(parser, field: Field, prefix: str = "") -> None:
+    if not _is_command(field):
+        raise TypeError("An action field must be a Union of dataclasses.")
```

Comment:
I think this is redundant because it's already wrapped in an if statement above

Reply:
True; removed!